### ISSUE TITLE
Unconditional dependency on futures breaks things on Python 3 as of futures 3.2.0

### ISSUE THREAD

**Issue by homeworkprod** [miguelgrinberg/Flask-MarrowMailer#3]
The futures package should no longer be installed on Python 3 at all.

This is detailed in the current version of the README: https://github.com/agronholm/pythonfutures/blob/master/README.rst

Please see my bug report here that adds to the discussion: https://github.com/agronholm/pythonfutures/pull/71#issuecomment-372037732

**Comment by miguelgrinberg** [comment]
Marrow mailer uses the futures package. If they stop requiring it I can remove it from here as well. See https://github.com/marrow/mailer#52-futures-manager.

**Comment by agronholm** [comment]
The requirement is already conditional on the Python version: https://github.com/marrow/mailer/blob/develop/setup.py#L70

**Comment by agronholm** [comment]
As you can see, installing marrow-mailer on Python 3.6 does **not** install the futures backport:
```
$ pip install marrow-mailer
Collecting marrow-mailer
  Downloading marrow.mailer-4.0.2-py2.py3-none-any.whl (132kB)
    100% |████████████████████████████████| 133kB 873kB/s 
Collecting marrow.util<2.0 (from marrow-mailer)
  Downloading marrow.util-1.2.3.tar.gz
Installing collected packages: marrow.util, marrow-mailer
  Running setup.py install for marrow.util ... done
Successfully installed marrow-mailer marrow.util-1.2.3
```

**Comment by agronholm** [comment]
The problem is not marrow-mailer, it is this library which requires it unconditionally. I don't understand why though, since this library doesn't even import `concurrent.futures`.

**Comment by homeworkprod** [comment]
Marrow-Mailer [requires futures itself, and on Python 2 only](https://github.com/marrow/mailer/blob/develop/setup.py#L70) (since more than two years ago). (Edit: Oh wait, that has already been pointed out, sorry.)

> If they stop requiring it I can remove it from here as well.

@miguelgrinberg, why did you add the explicit requirement in Flask-MarrowMailer at all if it's already required by Marrow-Mailer itself and (as @agronholm says) it's not used in Flask-MarrowMailer's code?

**Comment by miguelgrinberg** [comment]
Sorry, I misunderstood the problem. I agree futures should not be required. For Python 2 I included it to avoid people copying the documentation and examples and not getting the package installed, but yeah, Python 3 does not need it. Sorry about the confusion.

**Comment by miguelgrinberg** [comment]
Fixed in 0.3.0.

**Comment by homeworkprod** [comment]
Thank you for the quick fix and release! v0.3.0 solves the issue for me. Much appreciated!
